src/themes/create-theme.js

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright (c) 2018-2020 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import animation from './shared/animation.js';
+import borders from './shared/borders.js';
+import breakpoints from './shared/breakpoints.js';
+import darkBorders from './dark-theme/borders.js';
```

Comment:
As I mentioned in my previous comment we should only use as a base a single theme, whichever we consider as a default or base (currently it's LightTheme and it's also used as a default theme in the provider).
And with growing the theme base options the bundle size will also grow.

Reply:
I'll create a new function just for dark themes.